src/proto/single/mod.rs

```diff
@@ -206,19 +208,39 @@ impl JobBuilder {
     }
 }
 
+/// Details on a job's failure.
 #[derive(Serialize, Deserialize, Debug, Clone)]
+#[non_exhaustive]
 pub struct Failure {
-    retry_count: usize,
-    failed_at: String,
+    /// [`Number`](Job::retry) of times this job can be retried.
```

Comment:
Are we sure that this is the number of times it _can_ be retried, as opposed to the number of times it _has_ been retried (which it feels like the name implies)

Reply:
You are right and we are actually [demonstrating](https://github.com/jonhoo/faktory-rs/blob/feature/job-failure/tests/real/community.rs#L1054-L1058) this in the tests